### PR TITLE
add travis CI config to keywhiz-fs

### PR DESCRIPTION
**Pull request by riyazdf**
Enable Travis by adding yml file, runs tests on pull requests and pushes

### CODE REVIEW

**Comment by riyazdf** [comment]
R: @sul3n3t

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+sudo: true
+
+go:
+- 1.4
+- tip
+
+before_script:
+- go build keywhizfs/main.go
+- sudo apt-get install fuse-utils
```
Comment (sul3n3t) [inline .travis.yml]: Is installing fuse-utils (and `sudo: true`) necessary since fs_test.go is excluded?

Comment (riyazdf) [inline .travis.yml]: I'd still receive an error message about fusermount even after excluding fs_test.go  (example here: https://travis-ci.org/square/keywhiz-fs/builds/56611797)

Comment (sul3n3t) [inline .travis.yml]: Aha, I see.

**Comment by sul3n3t** [comment]
:+1:

**Pull request merged**